pyodata/v2/service.py

```diff
@@ -1392,7 +1410,12 @@ def create_entity_handler(response):
 
             entity_props = response.json()['d']
 
-            return EntityProxy(self._service, self._entity_set, self._entity_set.entity_type, entity_props)
+            try:
+                etag = response.headers.get('ETag', None)
+            except AttributeError:
```

Comment:
For the record, please explain this AttributeError catch statement or remove it.

Reply:
This can just be removed because .get will never raise an AttributeError??